### DISCUSSION TITLE
FTDI device acting as a SPI slave

### DISCUSSION THREAD

**Discussion by andreiung** [eblot/pyftdi#324] [Q&A]
Is there any FTDI device that can act as SPI slave ? Does the library pyftdi supports a SPI slave interface?

**Comment by eblot** [comment]
More or less. 

* Some FTDI acts as slaves, but as far as I understand you cannot manage/define the protocol, and they look like acting like basic SPI slave to USB bridge. They cannot be driven by software, but are quite limited in what they can do/what they accept on the SPI side, see [FT221](https://ftdichip.com/products/ft221xs/) series. There is no need for PyFtdi, as the device cannot be driven.
* [FT4222](https://ftdichip.com/products/ft4222h/) has more feature, but cannot neither be driven by PyFtdi, since it does not support the MPSSE protocol.

PyFtdi drives FTDI chips that use the MPSSE protocol, i.e. FT*232* devices.
